### PR TITLE
(issue-2):Setup an empty react project

### PR DESCRIPTION
**Pull request by ekta1998-cell**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "trackoice",
+  "private": true,
+  "version": "0.0.0",
```
Comment (gagandeep-singh-sde) [inline package.json:4]: Set the appropriate version

File: .gitignore
Comment (gagandeep-singh-sde) [inline .gitignore:1]: Only ignore the relevant types, remove the types which are not needed for this project

File: README.md
Comment (gagandeep-singh-sde) [inline README.md:1]: You have overwritten the readme file. Change the readme file appropriately.

**Comment by gagandeep-singh-sde** [comment]
Good work! But please resolve the comments and give attention to details next time onwards.

File: .gitignore
Comment (ekta1998-cell) [reply .gitignore:1]: hi , I am not sure what types are not relevant

**Pull request merged**